data/siggraph-2019/10.1145-3306346.3322986.json

```diff
@@ -0,0 +1,203 @@
+        "##": "0 = unable to set the required dependencies",
+        "##": "1 = many dependencies required, difficult to set up, unspecified version numbers..",
+        "##": "5 = no dependencies or some dependencies but trivial build/integration with the project",
+        "Dependencies score {0=NA, 1,2,3,4,5}": 4,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the build/configuration stage",
+        "##": "0 = no instructions or unable to build",
+        "##": "1 = many issues during the build or configure",
+        "##": "5 = no problem nor issue during the build/configure",
+        "Build/configure score {0=NA, 1,2,3,4,5}": 4,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the bug fixes (if any)",
+        "##": "0 = no bugs or irrelevant",
+        "##": "1 = many bugs, unable to fix them",
+        "##": "5 = just very minor bugs but very easy to fix",
+        "Fixing bugs score (if any) {0=NA, 1,2,3,4,5}": 5,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the replicability",
+        "##": "0 = N/A",
+        "##": "1 = I was not able to generate something or unable to analyze the output",
+        "##": "5 = I was able to reproduce most paper results and the remaining ones could be trivially obtained",
+        "Replicate paper results score {0=NA, 1,2,3,4,5}": 2,
+
+        "##": "Score {0=NA, 1,2,3,4,5} for the adaptability",
+        "##": "0 = N/A",
+        "##": "1 = The code is very specific to the paper contribution and cannot be used in other projects",
+        "##": "5 = The code can be easily used for other projects (as it is) or can be easily extended to other contexts",
+        "Adaptability score to other contexts {0=NA, 1,2,3,4,5}": 5,
+
+        "##": "Time spent for the replicability test (from the download to the first successful run)",
+        "##": "0 = less than 10 min",
+        "##": "1 = between 10 and 20 min",
+        "##": "...",
+        "##": "10 = more than 100 min",
+        "Time spent for the test (code download to first successful run, [0,10], 10min slots, 100min max)": 2,
+
+        "##": "OS for the test",
+        "Operating system for the test": "macOS Catalina",
+
+        "##": "Detailed instructions (especially if you had to fix some bugs before being able to run a test)",
+        "Build instructions/comments": "The zip file contains a single c++ and a README with a g++ command line instruction. To avoid ubild issues (the given command line does not work on my macOS Catalina), I have created a small CMakeLists.txt (after renaming the harm_simple.cp to harm_simple.cpp): \n\nproject(harmonic)\nfind_package(CGAL REQUIRED COMPONENTS Core)\nadd_executable(harm_simple harm_simple.cpp)\ntarget_link_libraries(harm_simple CGAL::CGAL  CGAL::CGAL_Core)\n\n Once compiled, I was able to produce some optimized tetrahedral meshes but I was not able to check the results (the tetview viewer does not work anymore on macOS Catalina).",
```

Comment:
```suggestion
        "Build instructions/comments": "The zip file contains a single c++ and a README with a g++ command line instruction. To avoid build issues (the given command line does not work on my macOS Catalina), I have created a small CMakeLists.txt (after renaming the harm_simple.cp to harm_simple.cpp): \n\nproject(harmonic)\nfind_package(CGAL REQUIRED COMPONENTS Core)\nadd_executable(harm_simple harm_simple.cpp)\ntarget_link_libraries(harm_simple CGAL::CGAL  CGAL::CGAL_Core)\n\n Once compiled, I was able to produce some optimized tetrahedral meshes but I was not able to check the results (the tetview viewer does not work anymore on macOS Catalina).",
```

Reply:
oops, thx